src/Servers/Kestrel/Core/src/Middleware/Internal/LoggingStream.cs

```diff
@@ -145,32 +145,52 @@ private void Log(string method, ReadOnlySpan<byte> buffer)
                 return;
             }
 
-            var builder = new StringBuilder($"{method}[{buffer.Length}] ");
+            var builder = new StringBuilder();
+            builder.AppendLine($"{method}[{buffer.Length}]");
```

Comment:
nit

How hot is this code? Rather than formatting a string and then adding it to the string builder, shouldn't each part be added individually?

e.g.
```cs
builder.Append(method);
builder.Append("[");
builder.Append(buffer.Length);
builder.Append("]");
builder.AppendLine();
```